### PR TITLE
Kyan 219 update kyanite with bulma v1

### PR DESCRIPTION
**Pull request by paweljozwicki**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue) labeled with `bug`
- [ ] New feature (non-breaking change which adds functionality) labeled with `enhancement`
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the [code standards](/CONTRIBUTING.md) of this project.
- [ ] My change requires updating the documentation. I have updated the documentation accordingly.

### CODE REVIEW

File: applications/luna-low-code/frontend/src/_css-variables.scss [hunk 1]
```diff
@@ -25,6 +25,7 @@
     --kyanite-primary-90-l: 92%;
     --kyanite-navbar-height: 6.25rem;
 
+    --color-primary--500: #1D1F31;
```
Comment (martyna-ds) [inline applications/luna-low-code/frontend/src/_css-variables.scss:28]: Is this some leftover? Why it is the same as `--color-default`, do we use it?

File: applications/luna-low-code/frontend/src/_css-variables.scss [hunk 2]
```diff
@@ -18,14 +18,15 @@
...
+    --kyanite-primary-l: 17%;
+
+    --kyanite-primary-90-l: 92%;
+    --kyanite-navbar-height: 6.25rem;
 
     --color-default: #1D1F31;
     --color-body-background: #FFFFFF;
-    --color-neutral-800: #151515;
-    --color-neutral-500: #727272;
-    --color-neutral-200: #F0F0F0;
```
Comment (martyna-ds) [inline applications/luna-low-code/frontend/src/_css-variables.scss:28]: Those neutral colors was never used?

File: applications/luna-low-code/frontend/src/_css-variables.scss (hunk 1 above)
Comment (paweljozwicki) [reply applications/luna-low-code/frontend/src/_css-variables.scss]: Yes, I don't know why it has the same value, so I changed the usages to --color-default and removed this value

File: applications/luna-low-code/frontend/src/_css-variables.scss (hunk 2 above)
Comment (paweljozwicki) [reply applications/luna-low-code/frontend/src/_css-variables.scss:28]: It looks like I removed too much :)

**Review by martyna-ds — APPROVED** [review]

**Pull request merged**